test/onyx/flow_conditions/flow_conditions_gen_test.clj

```diff
@@ -38,6 +38,12 @@
    :flow/from (gen/return from-task)
    :flow/exclude-keys (gen-sized-vector)))
 
+;(defn flow-short-circuit-condition-gen [from-task]
```

Comment:
Can you please delete this commented block? Otherwise this is great, will merge it when fixed!